corehq/blobs/migrate.py

```diff
@@ -240,6 +241,15 @@ def migrate(self, *args, **kw):
         hqmedia.CommCareVideo,
         hqmedia.CommCareMultimedia,
     ], CouchAttachmentMigrator),
+    Migrator("xforms", [
+        xform.XFormInstance,
+        ("XFormInstance-Deleted", xform.XFormInstance),
+        xform.XFormArchived,
+        xform.XFormDeprecated,
+        xform.XFormDuplicate,
+        xform.XFormError,
+        xform.SubmissionErrorLog,
```

Comment:
@dannyroberts: after searching in the code, it looks like `HQSubmission` may be a historical artifact. Is it important to migrate their attachments to the blob db?